src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+spring.application.name=fruits-ecommerce
+spring.datasource.url=jdbc:postgresql://localhost:5432/fruitsecommerce
+spring.datasource.username=postgres
```

Comment:
it's not recommended to connect to DB using the root user, Let's create a new user instead for the application to connect with